docs/book/src/quick-start.md

```diff
@@ -65,25 +54,25 @@ Kubebuilder provides autocompletion support for Bash and Zsh via the command `ku
 Create a directory, and then run the init command inside of it to initialize a new project. Follows an example.
 
 ```bash
-mkdir $GOPATH/src/example
-cd $GOPATH/src/example
-kubebuilder init --domain my.domain
+mkdir -p ~/projects/guestbook
+cd ~/projects/guestbook
+kubebuilder init --domain my.domain --repo my.domain/guestbook
 ```
 
 <aside class="note">
-<h1>Not in $GOPATH</h1>
+<h1>Developing in $GOPATH</h1>
 
-If you're not in `GOPATH`, you'll need to run `go mod init <modulename>` in order to tell kubebuilder and Go the base import path of your module. 
+If your project is initialized within [`GOPATH`][GOPATH-golang-docs], the implicitly called `go mod init` will interpolate the module path for you.
+Otherwise `--repo=<module path>` must be set.
 
-For a further understanding of `GOPATH` see [The GOPATH environment variable][GOPATH-golang-docs] in the [How to Write Go Code][how-to-write-go-code-golang-docs] golang page doc.   
+Read the [Go modules blogpost][go-modules-blogpost] if unfamiliar with the module system.
 
 </aside>
 
 <aside class="note">
 <h1>Go package issues</h1>
 
-Ensure that you activate the module support by running `$ export GO111MODULE=on` 
-to solve issues as `cannot find package ... (from $GOROOT)`.
+If you see an error like "cannot find package ... (from $GOROOT)", you may need to explicitly run `$ export GO111MODULE=on`.
```

Comment:
This should definetely not be needed in go1.15. Is there any case where the default behavior (auto) does not work and is not solved by the prefered `go mod init` command?

Reply:
No I think this can be removed completely.